public/js/user.js

```diff
@@ -6,16 +6,16 @@ function initLoginButton() {
     $("#logout").css("display", "none");
   });
   $("#editSite, #newSite, #locateMe").click(function(){
-    legendGroups = 2;
-    repaintMarkers();
+    //legendGroups = 2;
+    //repaintMarkers();
     initLegend();
     if ($(this).attr("id") === "locateMe")
       locateMe();
     if ($(this).attr("id") === "editSite") {
       getMarkerPosition(nextSite, function (result) {
+        nextSite = (nextSite+1)%auth_site.length;
```

Comment:
It's not obvious what this line of code is doing. I'm assuming that it is automatically setting focus on the next marker for the user and looping back around to the first. A small comment above this line wouldn't hurt. 🙂

Reply:
That's exactly what its doing! I added the comment. :)